src/PrototypeRouteListener.php

```diff
@@ -83,7 +83,9 @@ public function onMergeConfig(ModuleEvent $e)
         }
 
         // Override default version of 1 with user-specified config value, if available.
-        if (isset($config['zf-versioning']['default_version'])) {
+        if (isset($config['zf-versioning']['default_version'])
+            && !is_array($config['zf-versioning']['default_version'])
```

Comment:
This should probably be `is_scalar` instead of `! is_array`.

(I recommend `is_scalar` and not `is_numeric`, as we may allow string values at some point: e.g., to accommodate semantic versioning, or to allow "project name" versioning, ala the GitHub API.)

Reply:
much better for sure! Done with the changes!